data/area/asgarnia/falador/falador.npc-spawns.toml

```diff
@@ -250,16 +250,16 @@ spawns = [
   { id = "cyrisus_falador", x = 2997, y = 3340, members = true },
   { id = "karma_the_chameleon", x = 3054, y = 3371, members = true },
   { id = "sir_vant_falador", x = 2960, y = 3350, members = true },
-  { id = "sheep_black", x = 2893, y = 3324 },
-  { id = "sheep_black", x = 2895, y = 3323 },
-  { id = "sheep_black", x = 2895, y = 3326 },
-  { id = "sheep_black", x = 2897, y = 3324 },
-  { id = "sheep_black", x = 2899, y = 3326 },
-  { id = "sheep_black", x = 2900, y = 3324 },
-  { id = "sheep_black", x = 2902, y = 3326 },
-  { id = "sheep_black", x = 2905, y = 3324 },
-  { id = "sheep_black", x = 2907, y = 3326 },
-  { id = "sheep_black", x = 2908, y = 3324 },
+  { id = "black_sheep", x = 2893, y = 3324 },
```

Comment:
The reason they were named the old way is so they both match `sheep*`, you'll need to modify Sheep.kt so the matcher is `sheep*,black_sheep*` and also tweak it so black wool can be obtained